e2e-tests/src/test/staking.rs

```diff
@@ -55,10 +53,10 @@ pub fn staking_era_payouts(config: &Config) -> anyhow::Result<()> {
     let sender = validator_accounts[0].clone();
     let connection = create_connection(node).set_signer(sender);
 
-    batch_endow_account_balances(&connection, &stashes_accounts, VALIDATOR_STAKE);
+    batch_endow_account_balances(&connection, &stashes_accounts, MIN_VALIDATOR_BOND);
```

Comment:
maybe we can add `TOKEN` more as we do below?

Reply:
Yes, makes sense.